src/ai/susi/DAO.java

```diff
@@ -259,6 +260,15 @@ public void run() {
         OS.protectPath(fiveStarSkillRating_per);
         OS.protectPath(fiveStarSkillRating_vol);
 
+        //Last 7 days skill usage storage
+        Path susi_skill_usage_dir = dataPath.resolve("skill_usage");
+        susi_skill_usage_dir.toFile().mkdirs();
+        Path skillUsage_per = susi_skill_usage_dir.resolve("skillUsage.json");
+        Path skillUsage_vol = susi_skill_usage_dir.resolve("skillUsage_session.json");
```

Comment:
Looks good, but how do we know it's only `7` days? Is that available in the `skillUsage`? Would it be possible to add a query parameter in the usage service for the `range`? What if I want the last 10 days? How would this have to change?